eox_core/api/v1/views.py

```diff
@@ -99,10 +105,22 @@ def get_user_query(self, request, query_params=None):
 
         return user_query
 
+    def get_serialized_user(self, request, user):
+        """
+        Utility to get the serialized user with the proper account visibility configurations.
+        """
+        admin_fields = getattr(settings, 'ACCOUNT_VISIBILITY_CONFIGURATION', {}).get('admin_fields', {})
```

Comment:
If you want, you can split this line, with a keyword argument in his own line. @anmrdz

Reply:
done